test/elasticsearch.ts

```diff
@@ -113,6 +113,13 @@ describe("elasticsearch", () => {
 
   describe("update_replicas", () => {
     it("should make one command to put new index settings", (done) => {
+      if (!require("../config").indices.replicas) {
+        // TODO: methods in lib/elasticsearch.ts behave differently depending on settings in config.yml.
```

Comment:
NIT: would be nice to go ahead and create a test config file to drive the unitests.